### PR TITLE
Feature/recent history

### PR DESCRIPTION
**Pull request by Aurora12**
https://www.notion.so/worldr/API-Most-recent-slice-of-data-6e7b740a7f504d6b915cabe820b3ea1f

### CODE REVIEW

File: store/storetest/post_store.go [hunk 1]
```diff
@@ -2954,3 +2956,94 @@ func testPostStoreGetDirectPostParentsForExportAfterBatched(t *testing.T, ss sto
...
+			p.Message = msg + strconv.Itoa(j)
+			_, err := ss.Post().Save(&p)
+			require.Nil(t, err)
+		}
+
+	}
+	return &channels, &uids
+}
+
+func testGetRecentPosts(t *testing.T, ss store.Store, s SqlSupplier) {
```
Comment (ygworldr) [inline store/storetest/post_store.go:3002]: Shouldn't this be in its own `*_test.go` file?

Comment (Aurora12) [reply store/storetest/post_store.go:3002]: The code this method is testing resides in post.go, so I'm just following the convention.

**Comment by Aurora12** [comment]
> There is a fair amount of code but never little tests. This makes me uneasy… Could you convince me it's fine?

Integration tests (i.e. those that do the actual http requests) require way more work because they are organized differently and we only have an ad-hoc way to add them for our own handles. It should be a separate task for the next week to create a proper scaffolding for them and then add the tests for all of the new handles (7 already).

I'm always creating tests for the db methods, but integration tests are missing, you are right. This is why we didn't catch the error with channel_id parameter.

File: store/storetest/post_store.go (hunk 1 above)
Comment (ygworldr) [reply store/storetest/post_store.go:3002]: Okay.

**Pull request merged**